test/CMakeLists.txt

```
@@ -0,0 +1,38 @@
+#
+# Distributed Linear Algebra with Future (DLAF)
+#
+# Copyright (c) 2018-2024, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+set(MPIRUN "" CACHE STRING "MPI run command.")
```

I'd probably namespace this variable, since it's a cache variable:
```suggestion
set(DLAF_FORTRAN_MPIRUN "" CACHE STRING "MPI run command.")
```